Added Eth Guides template and beginning of IPFS eth Guide (#178)
protocol/launchpad: ethguides/intro-ipfs-filecoin.md

---
id: ?

> title: Introduciton to IPFS and Filecoin

Reviewer: ```suggestion
title: Introduction to IPFS and Filecoin
```

date: 'July 24, 2022'
slug: ipfs-101

> description: Conceptual understanding of they why and what of IPFS and FIlecoin

Reviewer: ```suggestion
description: Conceptual understanding of the why and what of IPFS and FIlecoin
```

thumbnail: 'ipfs-logo.png'
length: 3 mins
tags: ['101', 'web3']
author: ProtocolLabs
avatar: ipfs-logo.jpg
---
<!-- Note: Above header will be changed to a JSON file -->

<Section name="1. Why were IPFS and Filecoin Created?">

## Properties of the Next Version of the Internet

> > _The Properties of the internet and it's technologies define our human rights_ Juan Benet 2022

Reviewer: I think it should be
```suggestion
> _The Properties of the internet and its technologies define our human rights_ Juan Benet 2022
```

Technologies like household automation have given women independence, and the internet has given mass accessibility to education, ideas, and collaboration on a global scale.

> Protocol Labs and it's network oragnization's goal is, through the development of tecnologies, to

Reviewer: ```suggestion
Protocol Labs and its network organization's goal is, through the development of tecnologies, to
```
